spec/services/templates_service_spec.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'fileutils'
+
+describe TemplatesService do
+  include FirstTimeDataGenerator
+  let!(:main_team) { create :team }
+  let!(:admin_user) { create :user }
+
+  describe '#update_project' do
+    context 'update templates project' do
+      it 'experiment is added to templates project' do
+        create(:user_team, user: admin_user, team: main_team)
+        seed_demo_data(main_team.created_by, main_team)
+        dj_worker = Delayed::Worker.new
+        Delayed::Job.all.each { |job| dj_worker.run(job) }
+        Delayed::Job.all.each { |job| dj_worker.run(job) }
```

Comment:
Why duplicate?

Reply:
First run generates another set of delayed jobs

Reply:
OK, then just add comment so it is known.

Reply:
Apparently it doesn't